x-pack/solutions/observability/plugins/apm/test/scout/ui/parallel_tests/global_setup.ts

```diff
@@ -11,12 +11,20 @@ import { opbeans } from '../fixtures/synthtrace/opbeans';
 import { servicesDataFromTheLast24Hours } from '../fixtures/synthtrace/last_24_hours';
 import { testData } from '../fixtures';
 
-globalSetupHook('Ingest data to Elasticsearch', async ({ apmSynthtraceEsClient }) => {
-  const opbeansDataGenerator: SynthtraceGenerator<ApmFields> = opbeans({
-    from: new Date(testData.OPBEANS_START_DATE).getTime(),
-    to: new Date(testData.OPBEANS_END_DATE).getTime(),
-  });
+globalSetupHook(
+  'Ingest data to Elasticsearch',
+  { tag: ['@ess', '@svlOblt'] },
+  async ({ apmSynthtraceEsClient, apiServices, log }) => {
+    await apiServices.fleet.internal.setup();
```

Comment:
Probably for ECH you don't need the Fleet setup, then it can be skipped with

```
  async ({ profilingSetup, apiServices, log, config }) => {
    if (!config.isCloud) {
      // Create APM agent policy via Fleet API
      await apiServices.fleet.internal.setup();
...
```